rama-http-types/src/request.rs

```diff
@@ -0,0 +1,107 @@
+use crate::body::Body;
+use http::{Extensions, HeaderMap, HeaderValue, Method, Uri, Version, request::Parts};
```

Comment:
I know IDE's don't help here, but try to use the `crate::` versions please so we only have these external imports in 1 location. For stuff we do not re-export you do of course use them directly.